tests/test_asset.py

```diff
@@ -133,8 +149,13 @@ def test_download_stac_asset(asset_mock2, requests_mock):
         },
     )
     with tempfile.TemporaryDirectory() as tempdir:
-        out_path = asset_mock2.download_stac_asset(
+        out_path = asset_mock_2.download_stac_asset(
             pystac.Asset(href=STAC_ASSET_HREF, roles=["data"]), tempdir
         )
         assert out_path.exists()
         assert out_path.name == "bsg-104-20230522-044750-90756881_ortho.tiff"
+
+
+def test_asset_repr(asset_mock):
+    representation = repr(asset_mock)
```

Comment:
Let's inline this variable

Reply:
I have redone this test to use only `auth_mock` and `asset_info`